apps/spruce/src/pages/image/index.tsx

```diff
@@ -1,37 +1,44 @@
+import styled from "@emotion/styled";
+import { sideNavItemSidePadding } from "@leafygreen-ui/side-nav";
 import { Link, useParams, Navigate } from "react-router-dom";
 import { SideNav, SideNavGroup, SideNavItem } from "components/styles";
 import { ImageTabRoutes, getImageRoute, slugs } from "constants/routes";
+import { size } from "constants/tokens";
+import { useFirstImage } from "hooks";
 import { getTabTitle } from "./getTabTitle";
+import { ImageSelect } from "./ImageSelect";
 
 const Image: React.FC = () => {
-  const {
-    [slugs.imageId]: imageId,
-    [slugs.tab]: currentTab = ImageTabRoutes.BuildInformation,
-  } = useParams<{
+  const { [slugs.imageId]: imageId, [slugs.tab]: currentTab } = useParams<{
     [slugs.imageId]: string;
     [slugs.tab]: ImageTabRoutes;
   }>();
 
-  if (!Object.values(ImageTabRoutes).includes(currentTab)) {
+  const { image: firstImage } = useFirstImage();
+
+  const displayedImage = imageId ?? firstImage;
```

Comment:
I think calling this `selectedImage` might be more straightforward
```suggestion
  const selectedImage = imageId ?? firstImage;
```

Reply:
Agree!